### PR TITLE
Change nVidia to NVIDIA and add corresponding redirect

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Changed "nVidia" to "NVIDIA"

## QA

- View the site locally in your web browser at: https://ubuntu-com-13898.demos.haus/certified/vendors/nVidia
- See that you are redirected and the url now reads `.../NVIDIA` instead
- Check that the capitalization is correct in the hero, vendor filter, and results table

## Issue / Card

Fixes# https://warthogs.atlassian.net/browse/WD-11476

## Screenshots

image

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

**Comment by mtruj013** [comment]
@carkod since you're more familiar with the certified api would you mind taking a look at this one?

File: redirects.yaml [hunk 1]
```diff
@@ -136,6 +136,7 @@ certified/iot/models/?: "/certified/iot"
 certified/soc/models/?: "/certified/socs"
 certified/why-certified/?: "/certified/why-certify"
 certified/devices/?: "/certified/iot"
+certified/vendors/nVidia/?: "/certified/vendors/NVIDIA"
```
Comment (carkod) [inline redirects.yaml:139]: Is this necessary? URLs should always be lowercased?

Comment (mtruj013) [reply redirects.yaml:139]: It was included in the ticket instructions 
image

**Review by carkod — APPROVED** [review]
There are [still instances of Nvidia](https://ubuntu-com-13898.demos.haus/certified?q=nvidia), but as I told Felicia this should be changed on Certification API side.

I think this should be good enough.

File: redirects.yaml (hunk 1 above)
Comment (carkod) [reply redirects.yaml:139]: Ok but https://ubuntu.com/certified/vendors/nVidia is already going to https://ubuntu.com/certified/vendors/NVIDIA, so technically it's uncessary?

**Comment by mtruj013** [comment]
> this should be changed on Certification API side.

Agreed! I assumed this was supposed to be a quick fix while it's updated there

File: redirects.yaml (hunk 1 above)
Comment (mtruj013) [reply redirects.yaml:139]: It doesn't seem like it is, the url remains `nVdia`
image

**Pull request merged**